code/_core/obj/item/clothing/head/hats/witchunter.dm

```diff
@@ -1,16 +1,16 @@
-/obj/item/clothing/head/hat/witchunter

```

Comment:
Same issue here is happening. See https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings on how to fix this.